setup.py

```diff
@@ -15,7 +15,7 @@
 """
 from __future__ import division, print_function
 
-DOCLINES = __doc__.split("\n")
+DOCLINES = (__doc__ or '').split("\n")
```

Comment:
I'd argue it's fairly idiomatic Python, as you can find examples of the same idiom in the [Python standard library](https://github.com/python/cpython/blob/master/Lib/multiprocessing/util.py#L160) and [Django](https://github.com/django/django/blob/master/django/contrib/admin/sites.py#L505), and probably several other large Python projects. It's short and concise.

If that doesn't sway you, what about moving the conditional operator inside of parens like `DOCLINES = (__doc__ if __doc__ else '').split("\n")`. That at least removes the ambiguity of what evaluates when.